docker-compose.yml

```diff
@@ -17,3 +17,4 @@ services:
       - 69:27017
     volumes:
       - ./database/data/db:/data/db
+    command: ["--bind_ip_all"]
```

Comment:
Not ready for merge to master with bind to all. This exposes the database to all incoming connections. Instead a set of whitelisted IPs should be used or all connections should be blocked. Doing this in docker requires creating a network interface which I haven't figured out yet.